src/libraries/System.Private.CoreLib/src/System/UInt128.cs

```diff
@@ -1091,6 +1091,11 @@ public static UInt128 Log2(UInt128 value)
         /// <inheritdoc cref="IDivisionOperators{TSelf, TOther, TResult}.op_Division(TSelf, TOther)" />
         public static UInt128 operator /(UInt128 left, UInt128 right)
         {
+            if ((right._lower == 0) && (right._upper == 0))
+            {
+                throw new DivideByZeroException();
```

Comment:
Should this extract the throw info a helper method?

Reply:
Yes, this is a good idea.